src/main/java/net/atos/zac/mailtemplates/model/Mail.java

```diff
@@ -32,4 +34,11 @@ public enum Mail {
     public Set<MailTemplateVariabelen> getVariabelen() {
         return variabelen;
     }
+
+    public static class MailUtil {
```

Comment:
Hier hoef je geen aparte class voor te maken. Deze methode kan gewoon in de Mail enum

Reply:
Aangepast